releasenotes-builder/src/test/java/com/github/checkstyle/templates/TemplateProcessorTest.java

```diff
@@ -229,6 +229,23 @@ public void testGenerateOnlyGitHub() throws Exception {
         assertFile("githubPageBreakingCompatibility.txt", MainProcess.GITHUB_FILENAME);
     }
 
+    @Test
+    public void testNoEmptyLineInGitHub() throws Exception {
+        final List<String> errors = MainProcess.runPostGenerationAndPublication(
+            createNotes(createAllNotes(), null, null, null, null), createBaseCliOptions()
+                .setGenerateGitHub(true).build());
+
+        Assert.assertEquals("no errors", 0, errors.size());
+
+        final String lines = getFileContents(new File("src/main/resources/com/github/checkstyle/"
+           + "templates/github_post.template"));
+        final String[] linesList = lines.split("\\r?\\n");
+        for (String line : linesList) {
+            Assert.assertFalse("line should not be empty", line.isEmpty());
```

Comment:
Please remove such strange validation on empty lines, it is not required.
All that you need is validation of githubPageAll.txt

Reply:
then how we will be able to test the empty lines? As run `runPostGenerationAndPublication` runs on breaking compatibility text file